spec/models/waste_carriers_engine/order_item_spec.rb

```diff
@@ -55,8 +56,20 @@ module WasteCarriersEngine
         expect(order_item.amount).to eq(2_500)
       end
 
-      it "should set the correct description" do
-        expect(order_item.description).to eq("changing carrier type during renewal")
+      context "when the journey is :renewal" do
+        let(:journey) { :renewal }
+
+        it "should set the correct description" do
+          expect(order_item.description).to eq("changing carrier type during renewal")
```

Comment:
I think we discussed about the possibility to have a unique description which will apply to both journeys so that we don't have to make these special cases, was that ruled out then?

Reply:
I'm not sure to be honest - I don't remember talking about this.

Reply:
I think that both Andrew and Emi would be good if we change this to a simple `changing carrier type` so let's ask them tomorrow :D